.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: github
+repo_token: YOURTOKENHERE
```

Comment:
I guess we need to fill this, no ?

Reply:
Thankfully, no. If we want to enable coveralls, we can leverage GitHub Actions directly. The case where we would need this is if we wanted to report to Coveralls manually.